backend/layers/business/business.py

```diff
@@ -607,8 +607,9 @@ def ingest_dataset(
         # Validate the URIs
         # TODO: This should be done in the IngestionManifest class
         for key, _url in manifest.model_dump(exclude_none=True).items():
-            parsed_url = urlparse(str(_url))
-            _url = parsed_url.geturl()
+            if key == "flags":
+                continue
+            _url = str(_url)
```

Comment:
cutting out the middle logic of converting to parsed_url only to get the url back out.